README.md

```diff
@@ -76,15 +76,15 @@ $ git clone --recurse-submodules https://github.com/rapidsai/rmm.git
 $ cd rmm
 ```
 
-Follow the instructions under "Create the conda development environment `cudf_dev`" in the
+Follow the instructions under "Create the conda development environment `rmm_dev`" in the
 [cuDF README](https://github.com/rapidsai/cudf#build-from-source).
```

Comment:
This link does not take me to a specific location in the readme. I think this is the link: https://github.com/rapidsai/cudf/blob/main/CONTRIBUTING.md#build-from-source  . I think they should just use the name `cudf_dev` -- same environment for both libraries.